segmenters/image/TorchObjectDetectionSegmenter/manifest.yml

```diff
@@ -1,4 +1,4 @@
-manifest_version: 1
+manifest_version: 2
```

Comment:
this should not change, what needs to be updated the version of the image, not the manifest